static/app/components/events/traceEventDataSection.tsx

```diff
@@ -432,6 +440,16 @@ export function TraceEventDataSection({
               value={state.display}
               onChange={opts => handleDisplayChange(opts.map(opt => opt.value))}
               options={[{label: t('Display'), options: getDisplayOptions()}]}
+              shouldCloseOnBlur
+              menuFooter={
+                copyableStackTrace && (
+                  <SectionedOverlayFooter>
+                    <Button size="xs" icon={<IconCopy />} onClick={copyRawStackTrace}>
+                      {t('Copy raw stack trace')}
+                    </Button>
+                  </SectionedOverlayFooter>
+                )
+              }
```

Comment:
Ideally, the dropdown would close after this button was clicked, though I couldn't find a state-less way to achieve this. It's a small thing, but lmk if anyone knows if this is easily achievable without needing to take control over the open state of the menu.